tests/test_github.py

```diff
@@ -100,6 +100,18 @@ def test_comment_unmatched_event(self):
         self.waitUntilSettled()
         self.assertEqual(0, len(self.history))
 
+    def test_review_event(self):
+        A = self.fake_github.openFakePullRequest('org/project', 'master', 'A')
+        self.fake_github.emitEvent(A.getReviewAddedEvent('approved'))
+        self.waitUntilSettled()
+        self.assertEqual(3, len(self.history))
```

Comment:
This looks good, but it's not obvious to me what the assert is testing (I see it looks similar to the other tests, but why history == 3 vs history ==1?)